src/main/java/org/openstreetmap/atlas/checks/commands/FlagStatisticsSubCommand.java

```diff
@@ -0,0 +1,619 @@
+        outputLines.add(totals);
+
+        return outputLines;
+    }
+
+    /**
+     * Converts a map of flag counts per check per country into a table structure with checks as the
+     * first column and total check counts as the second. All values are converted to strings for
+     * easy conversion to a printable csv string.
+     *
+     * @param countryCheckCounts
+     *            a 2D {@link Map} of flag {@link Counter}s per check {@link String} per country
+     *            {@link String}
+     * @return a 2D {@link List} of {@link String}s
+     */
+    private List<List<String>> generateTotalsOutputStage1(
+            final Map<String, Map<String, Counter>> countryCheckCounts)
+    {
+        // Get a list of check names in alphabetical order
+        final List<String> checks = countryCheckCounts.entrySet().stream()
+                .flatMap(entry -> entry.getValue().keySet().stream()).distinct().sorted()
+                .collect(Collectors.toList());
+        // A list to hold all the output strings as a table of lines
+        final List<List<String>> outputLines = new ArrayList<>();
+
+        // Generate the header row
+        final List<String> headers = new ArrayList<>();
+        headers.add(CHECK);
+        headers.add("Input");
+        outputLines.add(headers);
+
+        // For each check...
+        checks.forEach(check ->
+        {
+            final List<String> checkRow = new ArrayList<>();
+            // Add the check name
+            checkRow.add(check);
+            // Add the total count
+            checkRow.add(String.valueOf(getCheckTotal(countryCheckCounts, check)));
+            outputLines.add(checkRow);
+        });
+
+        return outputLines;
+    }
+
+    /**
+     * Appends the totals of two maps of flag counts per check per country to the output of
+     * {@link #generateTotalsOutputStage1(Map)}.
+     *
+     * @param stage1Output
+     *            a 2D {@link List} of {@link String}s representing a table of checks names and
+     *            total counts
+     * @param referenceCountryCheckCounts
+     *            a 2D {@link Map} of flag {@link Counter}s per check {@link String} per country
+     *            {@link String}
+     * @param diffCountryCheckCounts
+     *            a 2D {@link Map} of flag {@link Counter}s per check {@link String} per country
+     *            {@link String}
+     * @return a 2D {@link List} of {@link String}s
+     */
+    private List<List<String>> generateTotalsOutputStage2(final List<List<String>> stage1Output,
+            final Map<String, Map<String, Counter>> referenceCountryCheckCounts,
+            final Map<String, Map<String, Counter>> diffCountryCheckCounts)
+    {
+        // Copy the output lines list
+        final List<List<String>> outputLines = new ArrayList<>(stage1Output);
+
+        // Add new headers
+        outputLines.get(0).add(1, "Reference");
```

Comment:
How do you feel about indicating these fields are sum's in the headers. Maybe  `Sum(Reference)` and `Sum(Difference)` ?